packages/flutter/lib/src/material/slider.dart

```diff
@@ -507,6 +507,10 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
 
   @override
   void dispose() {
+    if (overlayEntry != null) {
```

Comment:
In range slider, you do this logic at the end of the dispose method, I'm not sure if the order is important but either way it should be consistent between both.

Reply:
Done